brainscore_vision/model_helpers/activations/core.py

```diff
@@ -33,16 +37,56 @@ def __init__(self, get_activations, preprocessing, identifier=False, batch_size=
         self._stimulus_set_hooks = {}
         self._batch_activations_hooks = {}
 
-    def __call__(self, stimuli, layers, stimuli_identifier=None):
+        self.number_of_trials = 1  # for use with microsaccades.
+        self.microsaccade_extent_degrees = 0.05  # how many degrees models microsaccade by default
+
+        # a dict that contains the image paths and their respective microsaccades. e.g.,
+        #  {'abc.jpg': [(0, 0), (1.5, 2)]}
+        self.microsaccades = {'pixels': {}, 'degrees': {}}
```

Comment:
I don't understand how the comment and the dict instantiation fit together. `'pixels'` and `'degrees'` are not image paths?

Reply:
Yes, oops. A number of outdated comments in this file.